shared/actions/config.js

```diff
@@ -84,7 +84,8 @@ const getExtendedStatus = (): AsyncAction => dispatch => {
 
 const registerListeners = (): AsyncAction => dispatch => {
   dispatch(GregorCreators.listenForNativeReachabilityEvents)
-  dispatch(GregorCreators.registerGregorListeners())
+  dispatch(GregorCreators.registerGregorListeners)
```

Comment:
i think maybe this should just be
```
GregorCreators.registerGregorListeners()
```

with no dispatch, still call the function though